helix-view/src/editor.rs

```diff
@@ -224,6 +224,9 @@ pub enum LineNumber {
     /// If focused and in normal/select mode, show relative line number to the primary cursor.
     /// If unfocused or in insert mode, show absolute line number.
     Relative,
+
+    /// Don't show line number
```

Comment:
Yeah so the reason this was not done initially is because I was planning to make gutters configurable:

```toml
gutters = ["diagnostics", "line_number"]
```

But I guess we could use this as a stop gap.

Reply:
Yes, I also thought about that but I wasn't sure because `diagnostics` is actually `diagnostics_or_breakpoints` (combined), so people could be confused that breakpoints won't show up when deactivating `diagnostics`.

From a user perspective it would be clearer if it was
```rust
gutters = ["diagnostics", "breakpoints", "line_number"]
```
and then inside the `diagnostics_or_breakpoints` gutter function we would apply the configuration.

Reply:
@archseer I've made gutters configurable.
Note: Based on my previous comment, I also included `breakpoints`. In addition to that, I chose the kebab-case name `line-numbers` which is more consistent to the `line-number` configuration. I used plural because a gutter displays multiple line numbers (as it does display multiple breakpoints).

Reply:
Thinking about it again: Maybe it's not a good idea to introduce so many gutters in the configuration because
1) There might be more gutters in the future (git signs, ...) and done would need to configure them for each new feature
2) The length of the array doesn't correspond to the number of gutters, so we really should have one gutter per array entry.

Based on these reasons, I will do what your original proposal was (introduce only "diagnostics" and "line-numbers"), but I will mention in the comments that "diagnostics" also includes other features like breakpoints.

Reply:
Update: done